.gitignore

```diff
@@ -20,6 +20,11 @@ Assets/InitTestScene*.unity.meta
 ehthumbs.db
 Thumbs.db
 
+# ===================================== #
+# Proto generation system               #
+# ===================================== #
+#Generated/
+bin/
```

Comment:
why is this in the .meta file of the test scene?

Reply:
I don't understand, this is in the `.gitignore`